mappings/net/minecraft/command/arguments/NbtPathArgumentType.mapping

```diff
@@ -53,28 +53,24 @@ CLASS net/minecraft/class_2203 net/minecraft/command/arguments/NbtPathArgumentTy
 		METHOD method_9366 get (Lnet/minecraft/class_2520;)Ljava/util/List;
 			ARG 1 tag
 		METHOD method_9367 getOrInit (Lnet/minecraft/class_2520;Ljava/util/function/Supplier;)Ljava/util/List;
-			ARG 1 tag
 			ARG 2 source
 		METHOD method_9368 put (Lnet/minecraft/class_2520;Ljava/util/function/Supplier;)I
-			ARG 1 tag
 			ARG 2 source
 		METHOD method_9369 getTerminals (Lnet/minecraft/class_2520;)Ljava/util/List;
 			ARG 1 start
 		METHOD method_9371 forEach (Ljava/util/List;Ljava/util/function/Function;)I
-			ARG 0 tags
+			ARG 0 nbts
```

Comment:
Collections of NBT can probably still be referred to as `tags` within scopes like this. It is more readable than `nbts` IMO.

Reply:
Nbt is like Moose. The collective term for Nbt is Nbt.